scripts/tests/test_api_report.py

```diff
@@ -5,7 +5,7 @@
 from ..api_report import diff_versions
 
 
-@pytest.mark.api_test
+@pytest.mark.skip(reason="version checkouts currently too disruptive")
 def test_working_pipeline():
     first = diff_versions("0.11.0", "0.12.0").split("\n")
     with open("scripts/tests/samples/sample_diff.txt") as f:
```

Comment:
Unrelated. Turning off this test as accidentally running is very disruptive to a local dev env